build/dependencies.props

```diff
@@ -1,8 +1,11 @@
 <Project>
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <CoreFxVersion>4.3.0</CoreFxVersion>
```

Comment:
Yup. Any new packages you need can be added to this project: https://github.com/aspnet/CoreCLR/blob/dev/tools/MirrorLatestCoreCLR/restore-projects/CoreFx.proj